mods/cnc/sequences/vehicles.yaml

```diff
@@ -215,7 +215,7 @@ mlrs:
 	turret:
 		Start: 32
 		Facings: 32
-	turret-2:
+	aim:
```

Comment:
The problem is that the reload time is so short it actually does not reload but shoot shoot shoot shoot rockets until the aircraft falls to the ground.